src/pages/Send/components/RecipientPicker/Tabs/Tabs.js

```diff
@@ -0,0 +1,58 @@
+// @flow strict
+
+import * as React from 'react'
+import { t } from 'ttag'
+
+import classNames from 'classnames'
+
+import tabsStyle from './tabs.m.scss'
+
+export type Tab = 'contacts' | 'wallets'
+export type TabClickHandler = (tabName: Tab) => any
+
+type Props = {|
+  activeTab: Tab,
+  onTabClick: TabClickHandler,
+|}
+
+const handleTabClick = (onTabClick: TabClickHandler, tabName: Tab) => (e) => {
```

Comment:
check that this component doesn't render too many times, otherwise better to move it in component scope